### PR TITLE
JR2-183: Filter jobs by company name

### PR DESCRIPTION
**Pull request by birom**

### CODE REVIEW

File: src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java [hunk 1]
```diff
@@ -1292,6 +1292,8 @@ public void shouldFilterByPermanentContractType() throws Exception {
             .andExpect(status().isOk())
             .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
             .andExpect(header().string("X-Total-Count", "2"))
+            .andExpect(jsonPath("$.[0].id").value(not(job1.getId().toString())))
```
Comment (michael-wirth) [inline src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java]: if all values should match a condition it's better to use `"$.*.id"`

File: src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java [hunk 2]
```diff
@@ -1318,6 +1320,36 @@ public void shouldFilterByTemporaryContractType() throws Exception {
...
```
Comment (michael-wirth) [inline src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java]: use `"$.*.id"`

File: src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java [hunk 3]
```diff
@@ -1318,6 +1320,36 @@ public void shouldFilterByTemporaryContractType() throws Exception {
...
+        jobService.save(job0);
+        jobService.save(job1);
+        jobService.save(job2);
+
+        restJobMockMvc.perform(get("/api/_search/jobs")
+            .param("companyName", "CERN"))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+            .andExpect(header().string("X-Total-Count", "1"))
+            .andExpect(jsonPath("$.[0].id").value(job1.getId().toString()))
```
Comment (michael-wirth) [inline src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java]: use `"$.*.id"`

File: src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java (hunk 1 above)
Comment (birom) [reply src/test/java/ch/admin/seco/service/job/web/rest/JobResourceIntTest.java]: done

**Pull request merged**